src/web/releases.rs

```diff
@@ -1446,29 +1393,75 @@ mod tests {
     }
 
     #[test]
-    fn authors_page() {
+    fn nonexistent_owner_page() {
+        wrapper(|env| {
+            env.fake_release()
+                .name("some_random_crate")
+                .add_owner(CrateOwner {
+                    login: "foobar".into(),
+                    avatar: "https://example.org/foobar".into(),
+                    name: "Foo Bar".into(),
+                    email: "foobar@example.org".into(),
+                })
+                .create()?;
+            let page = kuchiki::parse_html().one(
+                env.frontend()
+                    .get("/releases/random-author")
+                    .send()?
+                    .text()?,
+            );
+
+            assert_eq!(page.select("#crate-title").unwrap().count(), 1);
+            assert_eq!(
+                page.select("#crate-title")
+                    .unwrap()
+                    .next()
+                    .unwrap()
+                    .text_contents(),
+                "The requested crate does not exist",
```

Comment:
The message is still this because of our BlockBlacklistedPrefixes.

See: https://github.com/rust-lang/docs.rs/blob/e6b702bf8fd434da63108b0ee725ab6c3df1cb2c/src/web/routes.rs#L325

Reply:
@jyn514 Do you have any suggestions to solve the problem? Or is this acceptable?

Reply:
Sorry I misunderstood this blacklist routing process, I think this should be correct.

Reply:
This is fixed by https://github.com/rust-lang/docs.rs/pull/1326.

Reply:
Fixed testing issue.